### PR TITLE
feature-NXP-19989-full-hd-picture-rendition

### PR DESCRIPTION
**Pull request by mnixo**

### CODE REVIEW

File: nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/resources/OSGI-INF/imaging-service-contrib.xml [hunk 1]
```diff
@@ -24,11 +24,15 @@
...
 
     <pictureConversion id="Small" description="Small size"
-      maxSize="280" order="100" chainId="Image.Blob.Resize"
+      maxSize="560" order="100" chainId="Image.Blob.Resize"
       default="true" rendition="true" />
 
     <pictureConversion id="Medium" description="Medium size"
-      maxSize="550" order="200" chainId="Image.Blob.Resize"
+      maxSize="1000" order="200" chainId="Image.Blob.Resize"
+      default="true" rendition="true" />
```
Comment (efge) [inline nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/resources/OSGI-INF/imaging-service-contrib.xml]: You're changing the size for Small and Medium conversions. Why? What about backward compatibility for existing customers that like or want this size. How is that related to adding the new FullHD size?

Comment (efge) [inline nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/resources/OSGI-INF/imaging-service-contrib.xml]: The commit message says "tweak" but you're now effectively making the Small size into what was previously the Medium one. That's not a tweak.

Comment (mnixo) [inline nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/resources/OSGI-INF/imaging-service-contrib.xml]: [Ticket updated.](https://jira.nuxeo.com/browse/NXP-19989)

**Comment by efge** [comment]
👍

File: nuxeo-features/preview/nuxeo-preview-core/src/main/java/org/nuxeo/ecm/platform/preview/adapter/ImagePreviewer.java [hunk 2]
```diff
@@ -28,14 +29,30 @@
...
+import static org.nuxeo.ecm.platform.picture.api.ImagingDocumentConstants.PICTURE_VIEWS_PROPERTY;
+
 /**
  * @author Alexandre Russel
  */
 public class ImagePreviewer extends AbstractPreviewer implements MimeTypePreviewer {
 
+    protected Blob getContentBlob(Blob original, DocumentModel doc) {
+        if (doc.getProperty(PICTURE_VIEWS_PROPERTY) != null) {
+            MultiviewPicture multiviewPicture = doc.getAdapter(MultiviewPicture.class);
```
Comment (nelsonsilva) [inline nuxeo-features/preview/nuxeo-preview-core/src/main/java/org/nuxeo/ecm/platform/preview/adapter/ImagePreviewer.java]: you shouldn't need the previous check. Normally you use doc.getAdapter() and check if it's null
